### PR TITLE
AAP-15785-Revisions: Revisions based on editorial review

### PR DESCRIPTION
**Pull request by lmalivert**
Jira issue [AAP-15785](https://issues.redhat.com/browse/AAP-15785)

- These edits/revisions are based off of [Ingrid's editorial review](https://docs.google.com/document/d/1FbYuUrfWUZEzc0Xko2t82c9uxoSnXVPZhRX_NhQlK1w/edit#heading=h.pga2xksfc8t2) **Chapter 4 Configuring user access for your local automation hub - 4.2.1**  

- In order to have this completed in time for SME review, I did not incorporate any changes that required rewriting, title change, etc.

### CODE REVIEW

File: downstream/modules/hub/proc-create-content-developers.adoc [hunk 1]
```diff
@@ -2,24 +2,24 @@
 
 = Creating a new group for content curators
 
-You can create a new group in {HubName} designed to support content curation in your organization that contributes internally developed collections for publication in {HubName}.
+You can create a new group in {HubName} designed to support content curation in your organization. This group can contribute internally developed collections for publication in {HubName}.
 
-Use this procedure to create and edit a group and assign the required permissions to help content developers create namespaces and upload their internally developed collections to {HubName}.
+To help content developers create namespace and upload their internally developed collections to {HubName}, you must first create and edit a group and assign the required permissions.
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-create-content-developers.adoc:7]: Suggest rewording to: To help content developers create **a** namespace...

File: downstream/modules/hub/proc-create-super-users.adoc [hunk 2]
```diff
@@ -4,16 +4,16 @@
...
+* You must be a *Super user*.  
 
 .Procedure
 . Log in to your local {HubName}.
-. From the navigation panel, select menu:User Access[Users].
-. Select the user you want to be a super user to see the User details page.
-. Select *Super User* under User type.
+. Navigate to menu:User Access[Users].
+. Select the user that you want to make a super user. The *User details* for that user are displayed.
+. Under *User type*, select *Super User* under User type.
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-create-super-users.adoc:17]: This sentence has some duplication and one instance of "under user type" should be removed.

File: downstream/modules/hub/proc-create-content-developers.adoc [hunk 3]
```diff
@@ -2,24 +2,24 @@
...
 
-* You have administrative permissions in {HubName} and create groups.
+* You have administrative permissions in {HubName} and can create groups.
 
 .Procedure
 
 . Log in to your local {HubName}.
-. From the navigation panel, select menu:User Access[Groups] and click btn:[Create].
-. Enter *Content Engineering* as a *Name* for the group in the modal and click btn:[Create]. You have created the new group and the *Groups* page appears.
+. Navigate to menu:User Access[Groups] and click btn:[Create].
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-create-content-developers.adoc:16]: We decided that the correct form for this is: "From the navigation panel, select menu:item1[]..."

File: downstream/modules/hub/proc-create-super-users.adoc [hunk 4]
```diff
@@ -4,16 +4,16 @@
...
 
-* You have *Super user* permissions and can create users in automation hub.  
+* You must be a *Super user*.  
 
 .Procedure
 . Log in to your local {HubName}.
-. From the navigation panel, select menu:User Access[Users].
-. Select the user you want to be a super user to see the User details page.
-. Select *Super User* under User type.
+. Navigate to menu:User Access[Users].
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-create-super-users.adoc:15]: We decided that the correct form for this is: "From the navigation panel, select menu:item1[]..."

File: downstream/modules/hub/proc-create-users.adoc [hunk 5]
```diff
@@ -2,20 +2,20 @@
...
+After you create a user in {HubName}, you can give them permissions by adding them to groups. Each group that can access features in the system associated to the level of assigned permissions.
 
 .Prerequisites
 
 * You have *user* permissions and can create users in {HubName}.
 
 .Procedure
 . Log in to your local {HubName}.
-. From the navigation panel, select menu:User Access[Users].
+. Navigate to menu:User Access[Users].
```
Comment (dcdacosta) [inline downstream/modules/hub/proc-create-users.adoc:15]: We decided that the correct form for this is: "From the navigation panel, select menu:item1[]..."

**Review by dcdacosta — COMMENTED** [review]
A few minor changes needed; otherwise, LGTM!

**Comment by lmalivert** [comment]
Thanks for the review, @dcdacosta!

**Review by dcdacosta — APPROVED** [review]
LGTM!
